xml/cap_depl_azure.xml

```diff
@@ -330,27 +336,7 @@ kube-system   tunnelfront-595565bc78-j8msn        1/1    Running   0          6m
  Happy Helming!</screen>
 
  </sect1>
- <sect1 xml:id="sec.cap.aks-storage-class">
```

Comment:
It was requested in https://github.com/SUSE/doc-cap/issues/369 that we have storage class information for AKS (like we do for other platforms). Perhaps we can update the content instead if it's not fitting.

Reply:
I added a comment to the scf-azure-config.yaml file, do you think it still needs the explanation?

Reply:
I think it's probably better to leave it in since a section has higher visibility than a comment in the config file